src/help/zaphelp/contents/start/concepts/concepts.html

```diff
@@ -27,6 +28,8 @@
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td><a href="scanpolicy.html">Scan Policy</a></td><td></td></tr>
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td><a href="scope.html">Scope</a></td><td></td></tr>
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td><a href="sessionManagement.html">Session Management</a></td><td></td></tr>
+<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td><a href="structmods.html">Structural Modifiers</a></td><td></td></tr>
+<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td><a href="structparams.html">Structural Parameters</a></td><td></td></tr>

```

Comment:
The new entries should be added after the "spider" entry, to keep the entries in alphabetic order.